examples/persistent_homology_graphs.ipynb

```diff
@@ -0,0 +1,659 @@
+{
```

Comment:
<p>I wonder whether a typical user may be confused by VR vs SR persistence. What do you think about removing the SR persistence analysis and including something like "Note: giotto-tda also supports an approximate scheme for computing topological quantities known as <code>SparseRipsPersistence</code>.  This can be useful for speeding up the computation on large inputs."</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/giotto-ai/giotto-tda/pull/462/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='3'/>

Reply:
<p>I can avoid the digression on <code>SparseRipsPersistence</code>,  but I think I'd still like to keep it in the list as is to avoid treating it as a second-class citizen.</p>

Reply:
<p>I've now addressed this by reemoving sparse Rips from the code cells and adding the note you suggested.</p><p> </p>